### DISCUSSION TITLE
x509: cannot validate certificate for because it doesn't contain any IP SANs

### DISCUSSION THREAD

**Discussion by ichinm** [bluenviron/mediamtx#656] [Questions] [answered]
Hi,

Just tried this using docker. Thank you for this, it solved most of my issues.

I have another problem with proxied source:

x509: cannot validate certificate for because it doesn't contain any IP SANs

conf:

paths:
  pineway-sub:
    source: https://192.168....
    sourceOnDemand: yes

cert is issued to reo-link but I used IP. Not try host name "reo-link" yet, even it works I cannot use it because I have many of them.

Is there any way to disable validation?

Thanks,

**Comment by aler9** [comment, marked as answer]
moved here #665
